src/test/java/com/karankumar/bookproject/ui/statistics/util/StatisticsViewTestUtils.java

```diff
@@ -105,4 +109,15 @@ private static Book createMobyDickBook(PredefinedShelfService predefinedShelfSer
         book.setRating(RatingScale.EIGHT_POINT_FIVE);
         return book;
     }
+
+    private static Book createHobbitBook(PredefinedShelfService predefinedShelfService) {
+        PredefinedShelf readShelf = getReadShelf(predefinedShelfService);
+        Author author = new Author("J.R.R", "Tolkien");
+        Book book = new Book("The Hobbit", author, readShelf);
+        book.setBookGenre(BookGenre.FANTASY);
+        book.setNumberOfPages(2000);
+        book.setPagesRead(1000);
+        book.setRating(RatingScale.EIGHT);
+        return book;
+    }
```

Comment:
As this is very similar to `createMobyDickBook()`, we could refactor this to eliminate code duplication. 

We could have a method called createBook() that takes in a book title, an `Author` and a `PredefinedShelfService` as parameters. The `createHobbitBook()` and `createMobyDickBook()` can then call this method with the relevant arguments.

Reply:
Done with 35fb2bc6f9597a7ead91d1ab44e60f3aff7e8ac2